policykit/policyengine/module.py

```diff
@@ -0,0 +1,76 @@
+from django.db import models
+import json
+from policyengine.models import (
+    CommunityPlatform,
+    CustomAction,
+    Procedure,
+    Execution,
+)
+
+class PolicyFactory():
+
+    def create_builtin_procedure_templates():
+        """
+            Create procedute templates for a given platform
+        """
+
+        variables_for_simple_vote_procedure = {
```

Comment:
Can we move this into a Slack specific script? It's a one-off script that just needs to run once at database initialization. I think we have something like this already for loading into the db our default starterkits?

Reply:
Yes, it is obviously a Slack procedure. My concern is actually about the bigger picture: I am not sure whether we would like to put in these integration-specific Procedure class, such as SlackProcedure. 

The first approach: Each defines one part of the procedure templates. The generic `Procedures` is responsible for defining the core procedure logic (such as how to determine the victory of a vote) and `SlackProcedure` for dealing with notifying users and initiating votes. 

The second approach: each defines the whole procedure but differs in how generalizable these procedures are. The generic `Procedures` define procedure templates that are applicable to all integrations, while `SlackProcedure` defines procedure templates that are only applicable to Slack.

Reply:
I think we would like to give users more freedom in creating policies and the first approach seems to assume the notification and initialization part of a procedure are less important, which restricts users' freedom in creating policies. 

But for the second approach, I also feel it is hard to define a prior whether a procedure is only applicable to this platform, as this may also depend on the similarity between this platform and that platform.